Library/Homebrew/command.rb

```diff
@@ -0,0 +1,321 @@
+module Homebrew
+  class Command
```

Comment:
Is there anything else that can be committed before this? This is still a 496 line diff which isn't easy to review. If it's the minimum: no worries and we can work on reviewing it.